test/test_quantized.py

```diff
@@ -1394,6 +1394,42 @@ def test_batch_norm(self, X, Y_scale, Y_zero_point, qengine):
             quantize_ref = torch.quantize_per_tensor(float_ref, Y_scale, Y_zero_point, dtype_x)
             self.assertEqual(qy.int_repr().numpy(), quantize_ref.int_repr().numpy())
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=5,
+                                              min_side=1, max_side=32),
+                       qparams=hu.qparams()),
+           Y_scale=st.floats(0.2, 2.6),
+           Y_zero_point=st.integers(0, 5),
+           qengine=st.sampled_from(("qnnpack", "fbgemm")))
```

Comment:
I believe there is no qengine for this op. We don't have qnnpack implementation.

Reply:
ok, let's remove it.